standard.md

```diff
@@ -6,18 +6,17 @@
-**6. Mechanism for Extracting Data** |	If this project has non personally identifiable information there must be a mechanism for extracting or importing non personally identifiable information (PII) data from the system in a non-proprietary format.
-**7. Adherence to privacy and applicable laws** |	The project must state that it complies with relevant privacy laws, and all applicable international and domestic laws.
-**8. Adherence to standards & best practices** |	Projects must demonstrate some adherence to standards, best practices and/or principles. i.e. the principles for digital development
-**9. Do No Harm** | 	All projects must demonstrate that they have taken steps to ensure that the project anticipates, prevents and does no harm. 
-**9a) Privacy & Freedom of Expression** |	All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on privacy and freedom of expression where governments are believed to be using the project’s product or services for illegitimate or political purposes.
-**9b) Data Privacy & Security** |	Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from it’s collection, storage and distribution.
-**9c) Inappropriate & Illegal Content** |	Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials and mechanisms for detecting and moderating inappropriate/illegal content.
-**9d) Protection from harassment** |	If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users.
+**6. Mechanism for extracting data** |	If this project has non personally identifiable information there must be a mechanism for extracting or importing non personally identifiable information (PII) data from the system in a non-proprietary format.
+**7. Adherence to privacy and applicable laws** |	The project must state that it complies with relevant privacy laws, and all applicable international and domestic laws. Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from its collection, storage and distribution.
+**8. Adherence to standards & best practices** |	Projects must demonstrate adherence to standards, best practices and/or principles. i.e. the principles for digital development.
```

Comment:
Thanks @sgoggins! One thing is that anything in this standard needs to be equally applicable to content, data sets, AI models so we likely can't get that granular. We're also keen that as much as possible have all of the indicators have objective yes/no requirements so that it can operate as a minimum standard. 

However I appreciate that principles have more longevity and are a better tool for helping projects improve over their development. One thing that could be cool would be to share principles with projects after submission as resources to helping them continue to improve. We could even send software specific to software, content to content etc.